src/pages/study-room/hook/useAudio.ts

```diff
@@ -0,0 +1,10 @@
+import { useRef } from 'react';
+import sound from '../../../assets/audio/flipdish-ringer.mp3';
+const useAudio = () => {
+  const audio = useRef(new Audio(sound));
+  const playSound = () => audio.current.play();
```

Comment:
useCallback으로 래핑해줘도 좋을 것 같습니다~

Reply:
이미 play라는 메소드가 audio에서는 익숙하기 때문에 그대로 play로 표현해주는건 어떨까요?
```suggestion
  const play = () => audio.current.play();
```